testsuite/tests/kuadrant/authorino/operator/http/conftest.py

```diff
@@ -0,0 +1,48 @@
+"""Conftest for all tests requiring custom deployment of Authorino"""
+import pytest
+from weakget import weakget
+
+from testsuite.openshift.objects.authorino import AuthorinoCR
+from testsuite.objects import Authorino, Authorization
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.httpx import HttpxBackoffClient
+
+HTTP_DEFAULT_PORT=5001
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization(authorization, authorino, wildcard_domain, blame, openshift, label, rhsso_service_info) -> Authorization:
+    """In case of Authorino, AuthConfig used for authorization"""
+    #if authorization is None:
+    authorization = AuthConfig.create_instance(openshift, blame("ac"), wildcard_domain, labels={"testRun": label})
+    authorization.add_oidc_identity("rhsso", rhsso_service_info.issuer_url())
+    authorization.add_response(inject_json={'name': 'another-json-returned-in-a-header',
```

Comment:
I might be missing something, but `add_response` doesn't seem to have a parameter called `inject_json`